diesel/src/pg/types/mod.rs

```diff
@@ -280,19 +280,73 @@ pub mod sql_types {
     /// ```
     #[derive(Debug, Clone, Copy, Default)] pub struct Money;
 
+    #[cfg(feature = "network-address")]
     /// The [`MACADDR`](https://www.postgresql.org/docs/9.6/static/datatype-net-types.html) SQL type. This type can only be used with `feature = "network-address"`
     ///
     /// ### [`ToSql`](/diesel/types/trait.ToSql.html) impls
     ///
-    /// - [`hwaddr::HwAddr`][HwAddr]
+    /// - `[u8; 6]`
     ///
     /// ### [`FromSql`](/diesel/types/trait.FromSql.html) impls
     ///
-    /// - [`hwaddr::HwAddr`][HwAddr]
+    /// - `[u8; 6]`
     ///
-    /// [HwAddr]: https://docs.rs/hwaddr/0.1.2/hwaddr/struct.HwAddr.html
+    /// # Examples
+    ///
+    /// ```rust
+    /// # #![allow(dead_code)]
+    /// # #[macro_use] extern crate diesel_codegen;
+    /// # #[macro_use] extern crate diesel;
+    /// # include!("src/doctest_setup.rs");
+    /// #
+    /// # table! {
+    /// #     users {
+    /// #         id -> Serial,
+    /// #         name -> VarChar,
+    /// #     }
+    /// # }
+    /// #
+    /// extern crate ipnetwork;
+    /// use diesel::types::MacAddr;
```

Comment:
Can you hide that ? I don't think we need the user where to get diesel types and it makes the examples less readable